packages-common/pluggable-widgets-generator/generators/app/templates/pluggable/native/emptyTemplateTs/src/WidgetName.tsx.ejs

```diff
@@ -0,0 +1,16 @@
+import { Component, ReactNode, createElement } from "react";
+import { TextStyle, ViewStyle } from "react-native";
+import { HelloWorldSample } from "./components/HelloWorldSample";
+import { <%= widgetName %>Props } from "../typings/<%= widgetName %>Props";
```

Comment:
We need to agreed with patterns, or we use <%= or <%- all around the code.

Reply:
Discussed. Let's use `<%- blah -%>`

Reply:
Done